js/ui/sessionMode.js

```diff
@@ -109,6 +109,40 @@ const _modes = {
             right: ['a11y', 'keyboard', 'aggregateMenu', 'userMenu',
                     'dateMenu', 'hotCorner']
         }
+    },
+
+    'user-coding': {
```

Comment:
Right you are! But you're missing the code that uses it then :-)
https://github.com/endlessm/eos-desktop/blob/master/js/ui/sessionMode.js#L181

Thinking more about it, I am not entirely sure we need a separate mode though, since we can have/already have settings for the individual components, and there's a cost in keeping these in sync.